f3.go

```diff
@@ -160,7 +156,7 @@ func (m *F3) Start(startCtx context.Context) (_err error) {
 	}
 
 	// Try to get an initial manifest immediately if possible so uses can query it immediately.
-	var pendingManifest *manifest.Manifest
+	var pendingManifest manifest.Manifest
 	select {
 	case pendingManifest = <-m.manifestProvider.ManifestUpdates():
```

Comment:
This will leave the manifest uninitialized.  We likely do need to keep the manifest nil in this case, or at least keep some kind of "have manifest" boolean.